install-dev/data/xml/hook.xml

```diff
@@ -741,5 +741,15 @@
       <title>Display new elements in back office product page, Combination tab</title>
       <description>This hook launches modules when the back office product page is displayed</description>
     </hook>
+    <hook id="displayDashboardToolbarTopMenu">
+      <name>displayDashboardToolbarTopMenu</name>
+      <title>Display new elements in back office products listing page, on top Menu</title>
+      <description>This hook launches modules when the back office product listing page is displayed</description>
+    </hook>
+    <hook id="displayDashboardToolbarIcons">
+      <name>displayDashboardToolbarIcons</name>
+      <title>Display new elements in back office products listing page, on icons list</title>
+      <description>This hook launches modules when the back office product listing page is displayed</description>
```

Comment:
Same comment about description

Reply:
In this case, the hook is really specific to the product catalog page, it's dispatched in `catalog.html.twig` template.

Reply:
Currently it is only dispatched from this file. But I'm pretty other pages will be migrated later and will display these small icons.
We won't come back to this hook description later, so I want it to fit its future purpose.